src/py/_pyodide/_core_docs.py

```diff
@@ -122,7 +125,13 @@ def create_proxy(obj: Any) -> JsProxy:
 
     # from python2js
 
-    def to_js(obj: Any, depth: int = -1) -> JsProxy:
+    def to_js(
+        obj: Any,
+        *,
+        depth: int = -1,
+        pyproxies: JsProxy = None,
+        create_pyproxies: bool = True
+    ) -> JsProxy:
```

Comment:
Generally we would need to add documentation for parameters,
```
Parameters
----------
obj : 
     Object to convert
depth : int, default=-1
     ...
```

and ideally a docstring example of how to use this. 

In `pyproxies` it would be good to mention that's it's a JsProxy of a JS Array.